cli/src/common.ts

```diff
@@ -304,9 +304,7 @@ export async function selectPlatforms(
       }
       fatal(
         `${c.strong(platformName)} platform has not been added yet.\n` +
-          `Use ${c.input(
-            `npx cap add ${platformName}`,
-          )} to add the platform to your project.`,
+          `https://capacitorjs.com/docs/v3/${platformName}#adding-the-${platformName}-platform`,
```

Comment:
Links should use `c.strong()` and have some context like "See these docs: <link>"